core/ts/__tests__/Poll.test.ts

```diff
@@ -407,7 +414,7 @@ describe("Poll", function test() {
     poll.publishMessage(message, ecdhKeypair.pubKey);
 
     it("should throw if called before all messages have been processed", () => {
-      expect(() => poll.tallyVotes()).to.throw("Cannot read properties of undefined (reading 'root')");
+      expect(() => poll.tallyVotes()).to.throw("Cannot convert undefined to a BigInt");
```

Comment:
It's really strange error from UX perspective. Is it possible to use try/catch to tell what actually is the problem?